tasks/run.mjs

```diff
@@ -12,10 +12,15 @@ export async function npmScript (name, args = []) {
   const command = process.platform === 'win32' ? 'npm.cmd' : 'npm'
 
   return new Promise((resolve, reject) => {
-    const script = spawn(command, ['run', name, ...args])
+    const script = spawn(command, ['run', name, '--silent', ...args])
```

Comment:
We still add `--silent` behind the scenes because we don't want both **npm** AND **Gulp** logging progress to the console—one is enough

**Note:** _Output_ from the npm task (linting for example) is not silenced